client/src/components/application/Examine/RequestInfoHeader.vue

```diff
@@ -712,6 +725,31 @@
       },
     },
     methods: {
+      async setNewExaminer() {
+        console.log('OMZAMANI-->setNewExaminer')
```

Comment:
maybe remove your idir from logs

Reply:
It's for posterity.

Reply:
also you know who to GITBlame when it all comes crashing down.